packages/react-dom/src/__tests__/ReactDOMSingletonComponents-test.js

```diff
@@ -996,17 +996,17 @@ describe('ReactDOM HostSingleton', () => {
   });
 
   // https://github.com/facebook/react/issues/26128
-  it('(#26128) does not throw when rendering at body', async () => {
+  it('(#26128) does not throw when rendering at body in legacy mode', async () => {
```

Comment:
We warn for this in the new root: https://github.com/facebook/react/blob/015ff2ed66c1d164111752263682d1d757c97f3e/packages/react-dom/src/__tests__/ReactDOMRoot-test.js#L224